src/Callback.php

```diff
@@ -155,12 +134,25 @@ protected function canTerminate(): bool
         return !$reload || $this->owner->name === $reload;
     }
 
+    public function isTriggered(): bool
+    {
+        if ($this->isPostTriggered) {
+            return isset($_POST[$this->urlTrigger]);
+        }
+
+        return isset($_GET[$this->urlTrigger]);
+    }
+
     /**
-     * Is callback triggered?
+     * Return this callback mode.
      */
-    public function triggered()
+    public function getMode(): string
```

Comment:
`getTriggeredMode` may be more contextual name

Reply:
Good point. I have changed it for getTriggeredValue.